extensions/macos-utilities/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "macos-utilities",
+  "title": "macOS Utilities",
+  "description": "A group of command items that are accessed via the command line like toggling clamshell mode, seeing your history and many many more...",
+  "icon": "icon.png",
+  "author": "kciarnie",
+  "categories": [
+    "Productivity",
+    "Applications",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "terminal-history",
+      "title": "Search Terminal History",
+      "description": "Search your terminal history, copy and paste the command into your terminal",
+      "mode": "view"
```

Comment:
Just my opinion, but terminal history management (great idea!) doesn't seem particularly macOS specific, for a "macOS Utilities" plugin.

Conversely, future features like toggling clamshell mode absolutely are macOS utility style tools, and I could see users wanting that functionality and not caring about terminal history.

Do you think this is perhaps two separate extensions? One for terminal history manipulation, and one for macOS utilities/tweaks/etc? They don't feel particularly related to me, and both feel like very useful extensions in their own right!

Reply:
Right now, I am planning on adding those other tools. I could easily keep this as one extension, but maybe @pernielsentikaer  can chime in here.

The thing is that I want to make sure it gets proper exposure in the right fields. I'm totally indifferent to where the extension goes.

Reply:
I might agree with @danpalmer since we already have [Shell](https://www.raycast.com/asubbotin/shell) in the Store which also reads the history, maybe it would be better to try an enhancement of that instead?

Reply:
Shall I still keep this extension here and remove the shell-history portion from here and add in the show/hide dock, toggle clamshell mode and other macOS specific items?

Reply:
It's always hard to figure out what the best approach is since we also have [Dock Tinker](https://www.raycast.com/koinzhang/dock-tinker) and [Dock](https://www.raycast.com/pradeepb28/dock) in the store

Reply:
Ahh okay, yeah, I noticed a bug with the show/hide toggle dock in `Dock Tinker`. In this case, I can update the `Dock` extension.